### DISCUSSION TITLE
How to make condition in Javascript?

### DISCUSSION THREAD

**Discussion by Daezyyy** [Daezyyy/javascripttttt#8] [Q&A] [answered]
lets discussions!

**Comment by DillonAdriyan** [comment, marked as answer]
Tentu, berikut adalah cara-cara untuk membuat kondisi dalam JavaScript:

### Statement `if`:
Digunakan untuk mengeksekusi kode tertentu jika kondisi yang ditentukan benar.
```javascript
let umur = 25;

if (umur >= 18) {
    console.log("Anda memenuhi syarat untuk memilih.");
}
```

### Statement `else`:
Digunakan untuk mengeksekusi kode tertentu jika kondisi dalam statement `if` adalah salah.
```javascript
let umur = 17;

if (umur >= 18) {
    console.log("Anda memenuhi syarat untuk memilih.");
} else {
    console.log("Anda belum memenuhi syarat untuk memilih.");
}
```

### Statement `else if`:
Digunakan untuk menguji beberapa kondisi dalam satu statement `if`.
```javascript
let umur = 25;
let status = 'single';

if (umur >= 18) {
    console.log("Anda memenuhi syarat untuk memilih.");
} else if (status === 'single') {
    console.log("Anda belum memenuhi syarat untuk memilih.");
} else {
    console.log("Anda belum memenuhi syarat untuk memilih, tapi pasangan Anda bisa.");
}
```

### Statement `switch`:
Digunakan untuk melakukan aksi berbeda berdasarkan kondisi atau variabel yang berbeda.
```javascript
let nilai = 'B';

switch (nilai) {
    case 'A':
        console.log("Luar biasa!");
        break;
    case 'B':
        console.log("Bagus!");
        break;
    case 'C':
        console.log("Anda lulus.");
        break;
    case 'D':
        console.log("Coba lagi.");
        break;
    case 'F':
        console.log("Gagal.");
        break;
    default:
        console.log("Nilai tidak valid.");
}
```

Perlu diingat, ini hanya beberapa cara untuk membuat kondisi dalam JavaScript. Kamu bisa menggunakan kondisi-kondisi ini untuk mengendalikan alur logika di dalam kode.
